llmfoundry/data/finetuning/dataloader.py

```diff
@@ -390,7 +397,9 @@ def _download_remote_hf_dataset(remote_path: str, split: str) -> str:
             finetune_dir, f'.node_{dist.get_node_rank()}_local_rank0_completed')
         if dist.get_local_rank() == 0:
             try:
-                get_file(path=name, destination=destination, overwrite=True)
+                retry(num_attempts=num_retries, exc_class=ChunkedEncodingError)(
```

Comment:
not a blocker, but just fyi, In checkpoint downloading with OCI, we've seen different kinds of request exception. Let's see if there any other exception in this case.

Reply:
Sounds good, will add other network errors here